### ISSUE TITLE
Build via normal `go build` command

### ISSUE THREAD

**Issue by mcandre** [okta/okta-sdk-golang#66]
Can we please adhere to more standard Go build workflows, so that this library is easier for developers to build and install? Try to stick to pure `go build` ..., `go install` ... commands for development.

Curious if there are any pre-compilation steps like generating source code or obtaining other artifacts, that necessitate the current make-based workflow.

**Comment by ckelly-digicert** [comment]
Looking at the makefile, it appears that they are leveraging NodeJS to generate files for the build. However, I don't actually see a step where they actually "build" anything. They merely run `go fmt` on the generated files.

I'm running into the issue where a simple `go get` isn't cutting it with Go v1.12.4. They have been using dep, but now that Go Modules are a thing and will become the default with Go v1.13, I think it's time that this repo updated to fall in line.

**Comment by ckelly-digicert** [comment]
<update> As an update to my previous comment, I solved my `go get` issue. While `go get` was successfully pulling down the package, the import path needs to now be `github.com/okta/okta-sdk-golang/okta`.  i.e. Do this:
```
package main

import (
	"github.com/okta/okta-sdk-golang/okta"
)
```

Don't use the import used in the examples on the doc site, as they don't work. I'll test with an older version of Go that doesn't have Modules support to see if anything changes.

**Comment by bretterer** [comment]
We have moved the develop branch over to using `go.mod` for all future commits.  (#60)

We are in the process of working on a few items and will be releasing a 1.0.0 version of this sdk.

We use `make build` for building the sdk as most of the files contained in the sdk are generated from an openAPI spec.  We leave it to the enduser developer to decide if they want to actually build the repo down to a binary or anything of the likes.

as for the import paths, is this a documentation issue in how to use the library?  if so, this has been changed as well in https://github.com/okta/okta-sdk-golang/pull/61/files
